xarray/tests/test_datatree.py

```diff
@@ -666,6 +666,18 @@ def test_properties(self) -> None:
             "b": np.dtype("int64"),
         }
 
+    def test_dims(self) -> None:
```

Comment:
Should probably also add a similar test to the one you used for `DataArray` here too